src/app/zap-templates/zcl/basic-information-cluster.xml

```diff
@@ -0,0 +1,59 @@
+    <globalAttribute side="either" code="0xFFFD" value="3"/>
+
+    <attribute side="server" code="0x0000" define="INTERACTION_MODEL_VERSION" type="INT16U"      default="1"              writable="false" optional="false">InteractionModelVersion</attribute>
+    <attribute side="server" code="0x0001" define="VENDOR_NAME"               type="CHAR_STRING"             length="32"  writable="false" optional="false">VendorName</attribute>
+    <attribute side="server" code="0x0002" define="VENDOR_ID"                 type="INT16U"                               writable="false" optional="false">VendorID</attribute>
+    <attribute side="server" code="0x0003" define="PRODUCT_NAME"              type="CHAR_STRING"             length="32"  writable="false" optional="false">ProductName</attribute>
+    <attribute side="server" code="0x0004" define="PRODUCT_ID"                type="INT16U"                               writable="false" optional="false">ProductID</attribute>
+    <attribute side="server" code="0x0005" define="USER_LABEL"                type="CHAR_STRING" default=""  length="32"  writable="true"  optional="false">UserLabel</attribute>
+    <attribute side="server" code="0x0006" define="LOCATION"                  type="CHAR_STRING" default=""  length="16"  writable="true"  optional="false">Location</attribute>
+    <attribute side="server" code="0x0007" define="HARDWARE_VERSION"          type="INT16U"      default="0"              writable="false" optional="false">HardwareVersion</attribute>
+    <attribute side="server" code="0x0008" define="HARDWARE_VERSION_STRING"   type="CHAR_STRING"             length="64"  writable="false" optional="false">HardwareVersionString</attribute>
+    <attribute side="server" code="0x0009" define="SOFTWARE_VERSION"          type="INT32U"      default="0"              writable="false" optional="false">SoftwareVersion</attribute>
+    <attribute side="server" code="0x000A" define="SOFTWARE_VERSION_STRING"   type="CHAR_STRING"             length="64"  writable="false" optional="false">SoftwareVersionString</attribute>
+    <attribute side="server" code="0x000B" define="MANUFACTURING_DATE"        type="CHAR_STRING"             length="16"  writable="false" optional="true" >ManufacturingDate</attribute>
+    <attribute side="server" code="0x000C" define="PART_NUMBER"               type="CHAR_STRING"             length="254" writable="false" optional="true" >PartNumber</attribute><!-- TODO The actual length is not defined in the spec. But the current implementation expect a length -->
+    <attribute side="server" code="0x000D" define="PRODUCT_URL"               type="CHAR_STRING"             length="256" writable="false" optional="true" >ProductURL</attribute>
+    <attribute side="server" code="0x000E" define="PRODUCT_LABEL"             type="CHAR_STRING"             length="64"  writable="false" optional="true" >ProductLabel</attribute>
```

Comment:
See spec for lengths: the value for PART_NUMBER is now available.

```
| 12 | PartNumber              | <<string-def>>  | max 32     | F       | MS      | R V      | O
| 13 | ProductURL              | <<string-def>>  | max 256    | F       | MS      | R V      | O
| 14 | ProductLabel            | <<string-def>>  | max 64     | F       | MS      | R V      | O
| 15 | SerialNumber            | <<string-def>>  | max 32     | F       | MS      | R V      | O
```

Reply:
I have also updated again the format of the `basic-information-cluster.xml`, in order to try to makes it easier to spot differences with the spec.